apps/trackers/tests/test_common.py

```diff
@@ -691,6 +691,39 @@ def test_feedback_form(self, form_url):
             else:
                 assert "feedback form" not in tqb.description
 
+    @pytest.mark.parametrize("info_url", [None, "https://example.doc.com"])
+    def test_VM_Info_link(self, info_url):
```

Comment:
Again, I would discourage from using capital leters in snake case and I would make consistent usage of VM vs VulnMgmt. VM is commonly used acronym for Virtual Machine so I would avoid using it in this context. Also I think even the full name `vulnerability management` usage would not be wrong in this case.